File: 프로그래머스/lv1/12928. 약수의 합/약수의 합.java
class Solution {
    public int solution(int n) {
        int answer = 0;
        
        for(int i = 1; i <= n; i++) {
			if(n % i == 0) {
				answer += i;
			}
		} // i가 n의 약수일때 answer에 누적 합산한다
        return answer;
    }
}

File: 프로그래머스/1/12928. 약수의 합/약수의 합.js
function solution(n) {
    let answer = 0;
    
    for (let i = 1; i <= n; i++) {
        if (n % i === 0) {
            answer += i;
        }
    } // i가 n의 약수일때 answer에 누적 합산한다
    return answer;
}
